internal/exec/vendor_utils.go

```diff
@@ -369,8 +369,9 @@ func ExecuteAtmosVendorInternal(
 		if !useLocalFileSystem {
 			err = ValidateURI(uri)
 			if err != nil {
+				err = fmt.Errorf("invalid URI for component %s: %w", s.Component, err)
 				if strings.Contains(uri, "..") {
-					return fmt.Errorf("Invalid URI '%s': %w. Please ensure the source is a valid local path or a properly formatted URI.", uri, err)
+					return fmt.Errorf("%w: Please ensure the source is a valid local path", err)
```

Comment:
```suggestion
					return fmt.Errorf("%s: Please ensure the source is a valid local path: %w", s.Component, err)
```

Reply:
I already changed `err` to contain the component above this and thus just prepended it via `%w` to the string. Maybe I should use an `if ... else` instead to make it clearer?

I wasn't able to check how the result looks like due to #1027.